plugins/widget/plugin.js

```diff
@@ -1055,9 +1054,10 @@
 		 * {@link CKEDITOR.style#checkActive} method instead of using this method directly,
 		 * because unlike style's method, this one does not perform any checks.
```

Comment:
Not sure if this part should stay or not. Now this method checks much more than just classes.